### DISCUSSION TITLE
wezterm looses opacity after a while

### DISCUSSION THREAD

**Discussion by Dronakurl** [wezterm/wezterm#5914] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

gnome

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

I set 
```lua
config.window_background_opacity = 0.93
```
and get a nice opaque wezterm. After a while, the opacity disappears. I could not pin it down to what happens.
Does anyone have an idea how to log and find the source?

**Comment by Joe-273** [comment]
I encountered an issue with the opacity setting not working on Windows. 
It was working before, but for some reason, it suddenly stopped.

 My configuration is as follows:
 ```lua
return {
    window_background_opacity = 0.93
}
```
